bin/init.bash

```diff
@@ -264,6 +264,24 @@ set_fluentd_config() {
     replace_set_me "${file}" '.fluentd.forwarder.useRegionEndpoint' "${use_region_endpoint}"
 }
 
+# Usage: set_s3bucketalertscount_config <config-file>
+set_s3bucketalertscount_config() {
+    file=$1
+    if [[ ! -f "${file}" ]]; then
+        log_error "ERROR: invalid file - $file"
+        exit 1
+    fi
+    case ${CK8S_CLOUD_PROVIDER} in
+        citycloud)
+            yq4 --inplace '.prometheus.s3BucketAlerts.nrOfObjects.count = 1638400' "${file}"
```

Comment:
Maybe also enable the alert in these cases.

```suggestion
            yq4 --inplace '.prometheus.s3BucketAlerts.nrOfObjects.count = 1638400' "${file}"
            yq4 --inplace '.prometheus.s3BucketAlerts.nrOfObjects.enabled = true' "${file}"
```

Reply:
aaah, ofc! :+1: